src/index.ts

```diff
@@ -62,18 +62,18 @@ client.on('message', async (message: Message) => {
     if (!message.guild) return
 
     // Determine settings for this message
-    const savedGuild = await Guild.findByGuildId(message.guild.id)
-    const guildSettings = savedGuild?.settings || defaultSettings
+    //const savedGuild = await Guild.findByGuildId(message.guild.id)
+    //const guildSettings = savedGuild?.settings || defaultSettings
 
     // Ignore bots
     if (message.author.bot) return
 
     if (message.content.indexOf(config.prefix) !== 0) {
         // On messages without prefix run these commands
         // Randomly meow when a message is received
-        if (Math.random() < guildSettings.randomSpeechProbability) {
-            speak(message)
-        }
+        // if (Math.random() < guildSettings.randomSpeechProbability) {
+        //     speak(message)
+        // }
```

Comment:
Bring it back!

Reply:
Ah I deleted a lot of guild stuff on my mac so I didn't need mongo 👀